client/blocks/login/continue-as-user.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { parse as qsParse } from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { getCurrentUser } from 'state/current-user/selectors';
+import Gravatar from 'components/gravatar';
+
+/**
+ * Style dependencies
+ */
+import './continue-as-user.scss';
+
+function ContinueAsUser( { currentUser } ) {
+	const translate = useTranslate();
+	const [ validated, setValidated ] = useState( null );
```

Comment:
Can you use `/` as the initial value? It's later defaulted to `/` anyway...

Reply:
Done.